readme.md

```diff
@@ -133,4 +138,4 @@ Other lines are ignored, so it's safe to combine on-screen subtitles and VFD sub
 
 -----
 
-by akasaka, 2021~2022
\ No newline at end of file
+by akasaka, 2021~2023
```

Comment:
Thanks for this. The readme now could use a `contributors` section I guess :-)

Reply:
> Thanks for this. The readme now could use a `contributors` section I guess :-)

I'll leave this one to you, I think there are some cool image generators for contributors that updates automatically, I've never used them.